### ISSUE TITLE
Add Support for Importing Publications when Snowballing

### ISSUE THREAD

**Issue by josepaiva94** [blochberger/sokman#2]
Sorry to disturb again. As I can see in https://github.com/blochberger/sokman/blob/master/sok/management/commands/snowball.py#L157 snowball is not implemented?

**Comment by blochberger** [comment]
Yes, when snowballing, the identified publications cannot be imported, yet.

The DBLP API does not offer a lookup via DOI, hence heuristics have to be implemented. DOI lookup is offered through the website however, see https://dblp.uni-trier.de/lookup/.

**Comment by josepaiva94** [comment]
If I want to use only Semantic Scholar (already working I believe), can you describe what needs to be done?

**Comment by blochberger** [comment]
You should read the necessary values from the `obj` dictionary similar to `result` object in the the DBLP import:

https://github.com/blochberger/sokman/blob/c7848600ba3f61c55b25521e8f2c855a314cc1b2/sok/management/commands/dblpimport.py#L446-L484

You might need to determine a cite key heuristically and you should assign the `sok.models.SemanticScholar.paper_id`.
